Week3/homework/step2-6.js

```diff
@@ -4,15 +4,27 @@ const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
 function printArray2d(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  let matrix = '';
+  for (let i = 0; i < arr.length; i++) {
+    for (let j = 0; j < arr[i].length; j++) {
+      matrix += arr[i][j];
+    }
+  }
+  console.log(matrix);
 }
```

Comment:
You could console.log each element in the inner `for` loop directly. This would have passed the unit tests too:

```js
function printArray2d(arr) {
  for (let i = 0; i < arr.length; i++) {
    for (let j = 0; j < arr[i].length; j++) {
      console.log(arr[i][j]);
    }
  }
}
```